src/main/java/seedu/gamebook/logic/commands/HelpCommand.java

```diff
@@ -14,19 +14,18 @@
 public class HelpCommand extends Command {
 
     public static final HelpCommand DUMMY = new HelpCommand();
+    public static final HelpCommand ERROR = new HelpCommand("a");
```

Comment:
This variable seems like it's only used in test cases? I think it shouldn't be here since it is not used for any function within GameBook?